packages/istanbul-lib-instrument/test/varia.test.js

```diff
@@ -142,4 +142,13 @@ describe('varia', function () {
         code = v.getGeneratedCode();
         assert.ok(code.match(/cov_(.+);export class App extends/));
     });
+
+    it('does not add extra parenthesis when superclass is an identifier', function () {
+        var v = verifier.create('class App extends Component {};', { generateOnly: true }),
+            code;
+        assert.ok(!v.err);
+        code = v.getGeneratedCode();
+        console.log(code);
```

Comment:
let's get rid of this `console.log`.

Reply:
Oops, fixed with amended commit with writings on commit messages.